data/actionsmap/yunohost.yml

```diff
@@ -535,6 +535,31 @@ app:
                     full: --file
                     help: Folder or tarball for upgrade
 
+        ### app_change_url()
+        change_url:
```

Comment:
Don't know, we also have a `regen-conf`, I don't think that we have a very consistent naming scheme :x 

But that can be changed yes.